### ISSUE TITLE
reticulate crashes R session in RStudio 1.4 preview (1.4.993)

### ISSUE THREAD

**Issue by issactoast** [rstudio/rstudio#8285]
### System details

    RStudio Edition : Desktop
    RStudio Version :  1.4.993
    OS Version      :  Windows 10
    R Version       :  4.0.2 (2020-06-22)

### Steps to reproduce the problem

Run any python code in Rstudio

### Describe the problem in detail

![Untitled](https://user-images.githubusercontent.com/13303749/98153488-e45b8f00-1e98-11eb-8517-5ade4827a4de.png)

Inside of Rstudio, when I run the python code, the session crashes. Here is my py_config() info.

```
reticulate::py_config()
python: C:/Users/issac/miniconda3/envs/pytorch-playbook/python.exe
libpython: C:/Users/issac/miniconda3/envs/pytorch-playbook/python37.dll
pythonhome: C:/Users/issac/miniconda3/envs/pytorch-playbook
version: 3.7.9 (default, Aug 31 2020, 17:10:11) [MSC v.1916 64 bit (AMD64)]
Architecture: 64bit
numpy: C:/Users/issac/miniconda3/envs/pytorch-playbook/Lib/site-packages/numpy
numpy_version: 1.19.1
```

### Describe the behavior you expected

No crash session

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
[diagnostics-report (1).txt](https://github.com/rstudio/rstudio/files/5489798/diagnostics-report.1.txt)

**Comment by ronblum** [comment]
@issactoast Thank you for raising the issue! However, I can't reproduce it. Can you provide some sample code so that we can try to replicate the error?

**Comment by issactoast** [comment]
@ronblum I don't have any sample code about this. If I want to run any python code it immediately aborted. I don't have any problem with the current Rstudio Version 1.3.1093 about this issue.

does it related to the path of the R and Rstudio, and Conda? I have installed R and tools and Rstudio under the same folder; C:\R\ but the conda is in my user folder

![image](https://user-images.githubusercontent.com/13303749/98429710-fb070f00-206d-11eb-8ab5-be0df4817c93.png)

![image](https://user-images.githubusercontent.com/13303749/98429781-694bd180-206e-11eb-8e41-36523baf18f7.png)

![image](https://user-images.githubusercontent.com/13303749/98429799-87b1cd00-206e-11eb-84df-52c936033f3d.png)

I

**Comment by kevinushey** [comment]
Unfortunately, I don't think we'll be able to resolve this without being able to reproduce the issue locally.

**Comment by issactoast** [comment]
Hello, I figured out what makes the crash.

Set the Regional format as non-English. In my case,

Windows10 > Language > Regional format > Set as Korean

Please let me know the issue is reproducible.

**Comment by kevinushey** [comment]
Marking as 1.4-JR for investigation.

**Comment by ronblum** [comment]
I'm unable to reproduce this on Windows 10, even when switching to Korean, both when sourcing the script and running it line by line.

@issactoast The script I ran is very simple. Does it result in the error for you?

```
x = 5
print(x)
```

**Comment by shashashak** [comment]
I(also a Korean) have the same issue in Rstudio Version 1.4.1103 
 
![2021-01-12 (2)](https://user-images.githubusercontent.com/74488465/104294446-85eb0680-5502-11eb-8b45-dafdd6b563a8.png)

**Comment by issactoast** [comment]
@ronblum @shashashak I updated Rstudio to 1.4.1056 and also update R to 4.0.3, then the error was gone. Maybe @shashashak try to update your R version if it is not the 4.0.3

**Comment by ronblum** [comment]
@issactoast Thanks—good to know, and I'm glad it's working for you now!
@shashashak What version of R and operating system are you using?

**Comment by issactoast** [comment]
@ronblum Thanks for the quick reply always and appreciate all the hard work! I would like to mention this issue related to this encoding issue in the reticulate package. This is critical (maybe so much bothering..) for coding python in Rstudio with other non-Eng languages.

https://github.com/rstudio/rstudio/issues/8549

**Comment by shashashak** [comment]
@ronblum @issactoast 
I'm using Windows 10 and R 4.0.3

**Comment by Cytotoxicity** [comment]
@issactoast
I am also korean using R 4.0.3 and the latest version of Rstudio. In my case, same error occured despite I updated everything in my environmemt. Did you change your regional format?

**Comment by Cytotoxicity** [comment]
@issactoast @shashashak 
I solved the problem. Actually I don't know the update had the effect to solve it. First I updated R to the latest version(4.0.3) but the error wasn't gone. So I changed my regional format from Korean to English and finally the error was gone. Now it works well.

**Comment by ByungjunKim** [comment]
I also have the same problem.
I am using R with Korean.

**Comment by issactoast** [comment]
@Cytotoxicity Thanks, actually I have solved this issue with the same method. Update R to 4.0.3 and switch to English locale. still have errors with the Korean regional setting.

**Comment by issactoast** [comment]
![image](https://user-images.githubusercontent.com/13303749/106414332-5c930b80-641a-11eb-9587-ceea2f1c0c92.png)

@ronblum  I think this issue have been solved with #8549 pull. 

Now, I have tested python code in the Korean regional setting, no crash, and print Korean well!
